spring/src/test/java/org/axonframework/spring/eventsourcing/benchmark/JpaStorageEngineInsertionReadOrderTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2010-2022. Axon Framework
+ * Copyright (c) 2010-2023. Axon Framework
```

Comment:
I assume there's no equivalent of this in the `hibernate-6-integrationtests` module, as that module doesn't include Spring, right?

Reply:
Added one in `axon-spring-boot-3-integration-tests`.